docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -47,7 +47,7 @@ You can provide a custom configuration to Keycloak using a `ConfigMap`.
 The content of the `ConfigMap` is a JSON file based on the Keycloak export files.
 You can logically separate different configurations inside the `ConfigMap` by using different keys. However, each key *must* end in `.json`.
 
-The configuration will be read during the startup of Keycloak and loaded into the database.
+Changes in the underlying `ConfigMap` or `Secret` referenced by `customConfigurationRef` or `customEnvVariablesRef` will trigger an external Job to configure the Keycloak instance via its API. The configuration will be applied dynamically and loaded into the database without requiring a restart of the Keycloak instance.
```

Comment:
Maybe also note that you need to slap the claim a bit so crossplane actually reconciles stuff.